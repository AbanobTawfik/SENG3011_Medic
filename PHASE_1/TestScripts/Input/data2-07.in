#!/bin/sh
# empty timezones are ignored

start_date=2013-01-20T00:00:00
end_date=2013-01-20T07:59:59
timezone="   "
