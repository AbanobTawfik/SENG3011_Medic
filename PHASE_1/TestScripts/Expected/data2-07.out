{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "articles": [
        {
            "url": "url9",
            "reports": [
                {
                    "diseases": [],
                    "syndromes": [],
                    "event_date": "2013-01-20 00:00:00 to 2013-01-20 07:59:59",
                    "locations": []
                }
            ],
            "headline": "Headline of url9",
            "main_text": "This is the main text for url9.",
            "date_of_publication": "2013-01-30 12:00:00"
        }
    ]
}
